environments/conda-lock.yml

```diff
@@ -1819,60 +1819,6 @@ package:
       sha256: 52d3e6bcd442537e22699cd227d8fdcfd54b708eeb8ee5b4c671a6a9b9cd74da
     category: main
     optional: false
-  - name: black
```

Comment:
One fewer dependency! It does make me feel a little strange to leave `black` behind, I had bought in so hard to "just one formatter, deal with it, screw your style guide." But happy to have a new benevolent style dictator in my life :)